package.json

```diff
@@ -14,6 +14,7 @@
     "@near-wallet-selector/near-wallet": "^7.9.0",
     "@near-wallet-selector/neth": "^7.9.0",
     "@near-wallet-selector/sender": "^7.9.0",
+    "@near-wallet-selector/welldone-wallet": "8.0.0",
```

Comment:
Can you install the ^7.9.0 instead of 8.0.0 to make it consistent with the other package versions.